tools/vargo/src/util.rs

```diff
@@ -66,3 +66,65 @@ pub fn mtime_recursive(path: &Path) -> Result<FileTime, String> {
         .unwrap_or_else(|| FileTime::from_last_modification_time(&meta));
     Ok(max_meta)
 }
+
+pub fn commit_info() -> Result<String, std::io::Error> {
```

Comment:
Since this function's err value is going to be ignored by the caller, it is better to have it return an `Option<String>` rather than a `Result<String, ...>`

You'll have to change up the current `?` places to `.ok()?` to account for this change, but `Option` should better indicate what the intention of this function is.